net/acme-common/files/run.sh

```diff
@@ -306,17 +389,35 @@ issue_cert()
     fi
 
     if [ -e /etc/init.d/uhttpd ] && [ "$update_uhttpd" -eq "1" ]; then
-        uci set uhttpd.main.key="${domain_dir}/${main_domain}.key"
-        uci set uhttpd.main.cert="${domain_dir}/fullchain.cer"
+        if [ "$APP" = "uacme" ]; then
+            uci set uhttpd.main.key="${domain_dir}/private/${main_domain}.key"
+            uci set uhttpd.main.cert="${domain_dir}/fullchain.cer"
+        else
+            uci set uhttpd.main.key="${domain_dir}/${main_domain}.key"
+            uci set uhttpd.main.cert="${domain_dir}/fullchain.cer"
+        fi
         # commit and reload is in post_checks
     fi
 
     if [ -e /etc/init.d/nginx ] && [ "$update_nginx" -eq "1" ]; then
+        if [ "$APP" = "uacme" ]; then
+        sed -i "s#ssl_certificate\ .*#ssl_certificate ${domain_dir}/fullchain.cer;#g" /etc/nginx/nginx.conf
+        sed -i "s#ssl_certificate_key\ .*#ssl_certificate_key ${domain_dir}/private/${main_domain}.key;#g" /etc/nginx/nginx.conf
+        else
         sed -i "s#ssl_certificate\ .*#ssl_certificate ${domain_dir}/fullchain.cer;#g" /etc/nginx/nginx.conf
         sed -i "s#ssl_certificate_key\ .*#ssl_certificate_key ${domain_dir}/${main_domain}.key;#g" /etc/nginx/nginx.conf
+        fi
         # commit and reload is in post_checks
     fi
 
+    if [ -e /etc/init.d/haproxy ] && [ "$update_haproxy" -eq 1 ]; then
+        if [ "$APP" = "uacme" ]; then
+            cat ${domain_dir}/${main_domain}/cert.pem ${domain_dir}/private/${main_domain}/key.pem > ${domain_dir}/${main_domain}/full_haproxy.$
```

Comment:
Does uacme put the full certificate chain into cert.pem or is it only the issued cert?